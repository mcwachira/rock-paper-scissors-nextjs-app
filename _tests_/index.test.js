import Home from "../pages/index";
import "@testing-library/jest-dom";
import { fireEvent, render, screen } from "@testing-library/react";

// jest.mock("next/image", () => ({ src, alt }) => <img src={src} alt={alt} />)

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})


