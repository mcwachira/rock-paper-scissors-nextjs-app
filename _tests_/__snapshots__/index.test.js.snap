// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      Welcome to next
    </h1>
    <div
      class="scoreBoard"
    >
      <img
        alt="header icon"
      />
      <div
        class="score"
      >
        <h2>
          Score
        </h2>
        <span
          class="scoreValue"
        >
          0
        </span>
      </div>
    </div>
    <div
      class="container"
    >
      <div
        class="hands"
      >
        <div
          class="icon paper"
        >
          <img
            alt="paper image"
          />
        </div>
        <div
          class="icon scissors"
        >
          <img
            alt="scissors image"
          />
        </div>
        <div
          class="icon rock"
        >
          <img
            alt="rock image"
          />
        </div>
      </div>
    </div>
    <footer
      class="footer"
    >
      <div
        class="rules"
      >
        Rules
      </div>
    </footer>
  </div>
</div>
`;
